README.md

```diff
@@ -19,7 +20,44 @@ How to use:
 - Create or open a project
 - Run menu Example > Example Action, it should display a list of classes
 
+## Install UML Metamodel to Eclipse 
+
+Viatra engine requires the UML Metamodel to be able to define queries and execute transformations. Hence it is necessary to install UML Metamodel to Eclipse if someone would like to use/extend the _MD_plugin_skeleton_ project. On this page, we propose a generic workflow to install UML Metamodel as a _p2 repository_.
```

Comment:
In the previous section, VIATRA was written with all caps; here we should do the same.